assets/javascripts/discourse-assign/controllers/user-activity-assigned.js.es6

```diff
@@ -21,10 +23,25 @@ export default Ember.Controller.extend({
             ajax("/assign/unassign-all", {
               type: "PUT",
               data: { user_id: user.get("id") }
-            }).then(() => this.send("unassignedAll"));
+            }).then(() => this.send("changeAssigned"));
           }
         }
       );
+    },
+    unassign(topic) {
+      ajax("/assign/unassign", {
```

Comment:
hmmm isn't this ajax call elsewhere, should we not be importing a model of sorts and just calling it?